src/Libs/GdkWin32-4.0/Win32/Fence.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+
+namespace Win32;
+
+[UnsupportedOSPlatform("Linux")]
+[UnsupportedOSPlatform("OSX")]
```

Comment:
```suggestion
[SupportedOSPlatform("windows")]
```
* is more direct
* browser, and all other would need to be set as unspported otherwise too

Reply:
When the supported OS are linux, windows, and MacOS, then for
> with all 3 supported operating systems.
you don't need any annotation at all.
Then it's a matter of convention: whether annotate unsupported ones (deny-list like) or annotate when a type / member is specific for one OS (where the analyzer automatically flags all other OSes as unsupported).

> web assembly feel free to try. If you got it running let's talk about missing annotations
OK 🤣 (fun aside, I guess it could work by cross-compiling GObject to wasm, but for what use case?)